openquakeplatform_ipt/static/ipt/js/common.js

```diff
@@ -57,6 +52,121 @@ function gem_capitalize(string) {
     return string.charAt(0).toUpperCase() + string.slice(1);
 }
 
+var gem_ipt = {
+    check_exception: function(message) {
+        this.message = message;
+        this.name = "CheckException";
+    },
+
```

Comment:
Why this is called `check_exception`? From the name I would have expected some check on an exception, but this looks simply like a factory of exception objects. At this point I would call it just `exception`, since the usage is `throw new gem_ipt.exception(message) `